Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -1371,4 +1377,43 @@ public String getService () {
         key.append(getResources().getString(R.string.service));
         return key.toString();
     }
+
+    public void referrerData() {
+        InstallReferrerClient referrerClient;
+        referrerClient = InstallReferrerClient.newBuilder(this).build();
+        referrerClient.startConnection(new InstallReferrerStateListener() {
+            @Override
+            public void onInstallReferrerSetupFinished(int responseCode) {
+                try {
+                    switch (responseCode) {
+                        case InstallReferrerClient.InstallReferrerResponse.OK:
+                            System.out.println("INSIDE REFERRER RESPONSE SUCCESS ");
```

Comment:
Remove logs and add proper comment if you want to keep commented code, otherwise remove the commented code